t/rosa-svn-pre-commit/04-unicode.t

```diff
@@ -47,16 +47,13 @@ chmod +x repos/foo/hooks/pre-commit
 export LANG=C
 
 TEST_KEY="${TEST_KEY_BASE}-western"
-cat > rose-suite.info << '__INFO__'
-owner=ivy
-project=euro
-title=We should not éñçödê config files in latin-1/western
-__INFO__
-iconv -f UTF-8 -t LATIN1 rose-suite.info -o rose-suite.info
-# -----------------------------------------------------------------------------
```

Comment:
Has this been moved to a file due to portability issues of `iconv`?

Reply:
Can't remember TBH, would only have changed if the test failed.